source/gfoidl.Base64/ThrowHelper.cs

```diff
@@ -66,7 +66,8 @@ private static string GetResource(ExceptionRessource ressource)
             Debug.Assert(Enum.IsDefined(typeof(ExceptionRessource), ressource),
                 $"The enum value is not defined, please check the {nameof(ExceptionRessource)} enum.");
 
-            return s_resources.Value.GetString(ressource.ToString());
+            return s_resources.Value.GetString(ressource.ToString())
+                ?? throw new InvalidOperationException("Please file a issue -> https://github.com/gfoidl/Base64/issues/new as this should not happen");
```

Comment:
Also debug-assert, fail early during development, but make sure this is covered by a test.